{"@t":"2020-08-15T09:45:27.5338125Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.6.4.","TimingId":"2568528","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:27.5538182Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"CoffeeShop.Web","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"C:\\Users\\Reece\\source\\repos\\CoffeeShop\\CoffeeShop.Web\\","MachineName":"DESKTOP-8FP67KQ","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:27.6458385Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:27.6478392Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.2029660Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"6351a36","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.3539992Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":150,"TimingId":"6351a36","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.7250826Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.7250826Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.7260829Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.9551349Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"d653ecc","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.9691380Z","@mt":"Tried to load content from the local cache file but it was empty.","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.9691380Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":14,"TimingId":"d653ecc","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:28.9761386Z","@mt":"Loading content from local db raised warnings, will reload from database.","@l":"Warning","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.2071911Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"7e779ec","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.2071911Z","@mt":"Tried to load media from the local cache file but it was empty.","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.2071911Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":0,"TimingId":"7e779ec","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.2071911Z","@mt":"Loading media from local db raised warnings, will reload from database.","@l":"Warning","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.2121932Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from database","TimingId":"0031387","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.3562249Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":144,"TimingId":"0031387","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.3572255Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from database","TimingId":"42228a4","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.3922332Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":34,"TimingId":"42228a4","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.4482458Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.5652724Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.6042814Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.6232858Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.6242856Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":2098,"TimingId":"2568528","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"08fa545f-b0c2-4ac6-9524-ac0b9b998162"}
{"@t":"2020-08-15T09:45:29.9693640Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1b3292f7-faab-48d8-94f9-74acb4bc3c63"}
{"@t":"2020-08-15T09:45:29.9713647Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1b3292f7-faab-48d8-94f9-74acb4bc3c63"}
{"@t":"2020-08-15T09:46:47.5308936Z","@mt":"Umbraco has no content","@l":"Warning","SourceContext":"Umbraco.Web.UmbracoModule","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":20,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN ","HttpRequestNumber":3,"HttpRequestId":"72d137a5-d431-4b72-b74b-170ad173b66a"}
{"@t":"2020-08-15T09:47:09.4048381Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username admin@example.co from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"c1e4af10-563e-4522-ba27-18bfc6a49bdc"}
{"@t":"2020-08-15T09:47:09.4058378Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: admin@example.co logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"c1e4af10-563e-4522-ba27-18bfc6a49bdc"}
{"@t":"2020-08-15T09:49:57.3644248Z","@mt":"{LogPrefix} Task has failed","@l":"Error","@x":"System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\\Users\\Reece\\source\\repos\\CoffeeShop\\CoffeeShop.Web\\App_Data\\TEMP\\FileUploads'.\r\n   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)\r\n   at System.IO.FileSystemEnumerableIterator`1.CommonInit()\r\n   at System.IO.FileSystemEnumerableIterator`1..ctor(String path, String originalUserPath, String searchPattern, SearchOption searchOption, SearchResultHandler`1 resultHandler, Boolean checkHost)\r\n   at System.IO.DirectoryInfo.InternalGetFiles(String searchPattern, SearchOption searchOption)\r\n   at System.IO.DirectoryInfo.GetFiles(String searchPattern, SearchOption searchOption)\r\n   at Umbraco.Web.Scheduling.TempFileCleanup.CleanupFolder(DirectoryInfo dir) in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\TempFileCleanup.cs:line 56\r\n   at Umbraco.Web.Scheduling.TempFileCleanup.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\TempFileCleanup.cs:line 43\r\n   at Umbraco.Web.Scheduling.RecurringTaskBase.Run() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\RecurringTaskBase.cs:line 47\r\n   at Umbraco.Web.Scheduling.BackgroundTaskRunner`1.<RunAsync>d__40.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\BackgroundTaskRunner.cs:line 562","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T09:50:01.3713298Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T09:50:21.4398671Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"Debug compilation mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"WARN "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:50:21.4408666Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL https://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO "}
{"@t":"2020-08-15T09:55:05.3890425Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:00:09.3927705Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:05:13.4065002Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:06:40.1591075Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: admin@example.co, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":303,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"ec38a3ab-ba49-43aa-9a1f-528ae4c33c60"}
{"@t":"2020-08-15T10:10:17.4242207Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:15:21.4399232Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:20:25.4546438Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:25:29.4703550Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:30:33.4850664Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:35:37.5017688Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":338,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:40:41.5164803Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:45:45.5312385Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:50:49.5479503Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T10:54:07.0755855Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username admin@example.co from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":492,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"f196db24-4342-4241-b834-4419bb0eea97"}
{"@t":"2020-08-15T10:54:07.0755855Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: admin@example.co logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":492,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"f196db24-4342-4241-b834-4419bb0eea97"}
{"@t":"2020-08-15T10:55:53.5503163Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:00:57.5637568Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:06:01.5811171Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:11:05.5965365Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:16:09.6095863Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:21:13.6246258Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
{"@t":"2020-08-15T11:26:17.6422627Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":21380,"ProcessName":"iisexpress","ThreadId":151,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-8FP67KQ","Log4NetLevel":"ERROR"}
